signal_handler.c

```diff
@@ -175,26 +200,42 @@ PHP_FUNCTION(attach_signal)
 
 	/* Check if given parameter is callable */
 	#if ZEND_MODULE_API_NO <= 20060613
-		if (!zend_is_callable(handle, 0, &func_name)) {
+	if (!zend_is_callable(handle, 0, func_name)) {
```

Comment:
You probably didn't want to remove the '&' here.  zend_is_callable() can't populate func_name by reference with this call.